website/src/components/articles/article-content-footer.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+
+interface Props {
+  lastUpdated: string;
```

Comment:
Always readonly for new or touched interfaces, please. I will soon update the entire codebase so that readonly is always used.